### ISSUE TITLE
Does yat support sticky posts?

### ISSUE THREAD

**Issue by xietx1995** [jeffreytse/jekyll-theme-yat#50]
I want some of my articles to always show on the top. It could be done by changing the date of posts, but it makes no sense.

**Comment by jeffreytse** [comment]
You can do this by easily modifying the index page (i.e. index.html) simply to add your custom HTML content, such as the recommended articles. But this feature is interesting, I will take a consideration.

**Comment by xietx1995** [comment]
> You can do this by easily modifying the index page (i.e. index.html) simply to add your custom HTML content, such as the recommended articles. But this feature is interesting, I will take a consideration.

Yeah, it's great. Maybe adding a recommended articles area is a good idea! I think it is better than sticky posts. If it could be done by simply add an entry in the front matter, see `recommend: true`, it would be a nice feature.
